src/sentry/api/base.py

```diff
@@ -364,7 +365,12 @@ def paginate(
                 sentry_sdk.set_tag(
                     "query.per_page.grouped", format_grouped_length(per_page, [1, 10, 50, 100])
                 )
-                cursor_result = paginator.get_result(limit=per_page, cursor=input_cursor)
+                if isinstance(paginator, Paginator):
+                    cursor_result = paginator.get_result(
+                        limit=per_page, cursor=input_cursor, count_hits=count_hits
+                    )
+                else:
+                    cursor_result = paginator.get_result(limit=per_page, cursor=input_cursor)
```

Comment:
I'd say that it might be fine to let a paginator that doesn't support `count_hits` just fail here. The failure should be caught in tests, so that the developer is aware of the problem. This check here will silently ignore the problem, so they might not be aware that they're not getting hits in the end.

Reply:
got it, I'll make another pr to change this